tensorflow_lite_support/python/task/vision/image_classifier.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Image classifier task."""
+
+import dataclasses
+from typing import Optional
+
+from tensorflow_lite_support.python.task.core import task_options
+from tensorflow_lite_support.python.task.processor import processor_options
+from tensorflow_lite_support.python.task.processor.proto import bounding_box_pb2
+from tensorflow_lite_support.python.task.processor.proto import classifications_pb2
+from tensorflow_lite_support.python.task.processor.proto import image_classifier_options_pb2
+from tensorflow_lite_support.python.task.vision.core import tensor_image
+from tensorflow_lite_support.python.task.vision.core.pybinds import image_utils
+from tensorflow_lite_support.python.task.vision.pybinds import image_classifier as _image_classifier
+
+from tensorflow_lite_support.python.task.vision.core.utils import build_proto_base_options
+
+
```

Comment:
Please adopt the same conventions as https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/python/task/vision/image_embedder.py, i.e:

_ProtoImageClassifierOptions = image_classifier_options_pb2.ImageClassifierOptions
_CppImageClassifier = _pywrap_image_classifier.ImageClassifier